k9mail-library/src/main/java/com/fsck/k9/mail/K9MailLib.java

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
I'm not a big fan of making this a super object in terms of configuration. `ImapConfig` would be a good name for a class that could house the new stuff being added here.